finished-application/backend/access.ts

```diff
@@ -35,7 +35,7 @@ export const rules = {
       return true;
     }
     // 2. If not, do they own this item?
-    return { user: { id: session.itemId } };
+    return { user: { id: { equals: session!.itemId } } };
```

Comment:
for some reason must be prettier or eslint is removing the `!` when I save

Reply:
```
This assertion is unnecessary since it does not change the type of the expression.eslint@typescript-eslint/no-unnecessary-type-assertion
Forbidden non-null assertion.eslint@typescript-eslint/no-non-null-assertion
```